.github/workflows/integration-local.yml

```diff
@@ -44,3 +45,70 @@ jobs:
       - name: Deploy Contracts
         run: flow project deploy --network=emulator --update
 
+  cypress:
+      name: Kitty Items Cypress End to End tests
+      timeout-minutes: 10
+      runs-on: ubuntu-latest
+      container: cypress/browsers:node16.14.2-slim-chrome103-ff102 #use arm64 arch on Apple M1 chips when running locally
+      steps:
+        - name: Install build dependencies (mostly for sqlite on ARM)
+          run: apt-get update -y && apt-get install build-essential -y 
+
+        - name: Install Flow CLI
+          run: | 
+            sh -ci "$(curl -fsSL https://storage.googleapis.com/flow-cli/install.sh)" &&
+            echo "/root/.local/bin:/github/home/.local/bin" >> $GITHUB_PATH
+            echo $HOME &&
+            ls -l $HOME/.local/bin &&
+            chmod a+r $HOME/.local/bin/flow &&
+            ls -l $HOME/.local/bin &&
+            $HOME/.local/bin/flow version
+          # run: | 
+          #   sh -ci "$(curl -fsSL https://raw.githubusercontent.com/onflow/flow-cli/master/install.sh)" &&
+          #   echo "/root/.local/bin:/github/home/.local/bin" >> $GITHUB_PATH &&
+          #   echo $HOME &&
+          #   ls -l $HOME/.local/bin &&
+          #   chmod a+r $HOME/.local/bin/flow &&
+          #   ls -l $HOME/.local/bin &&
+          #   $HOME/.local/bin/flow version
+
+        - name: Checkout
+          uses: actions/checkout@v3
+        
+        - name: chmod subdirs to bypass lerna exec access permission issues
+          run: |
+            pwd &&
+            ls -l && 
+            chmod -R 777 api &&
+            chmod -R 777 web &&
+            chmod -R 777 cadence/tests &&
+            ls -l 
+
+        - name: Cypress run
+          uses: cypress-io/github-action@v4
+          with:
+            start: npm run dev:emulator
+            wait-on: http://localhost:3001
+
+        # - name: Print logs
+        #   if: always()
+        #   run: npx pm2 logs    
```

Comment:
Can we remove the commented sections in this file?

Reply:
Absolutely, removed and updated PR